clientresources.json

```diff
@@ -1,29 +1,186 @@
+    },
+    "TotalDeathsOnaDay": {
+      "title": "Total Deaths cases on a particuar day",
+      "expression": "deaths(day)"
+    },
+    "TotalDeathsOnaDayPer1M": {
+      "title": "Total Deaths cases on a particuar day per million",
+      "expression": "deaths(day)/population*1000000"
+    },
+    "Last7DaysAverageCases": {
+      "title": "Average cases in Last 7 Days",
+      "expression": "average(cases(day-7,day))"
+    },
+    "Last7DaysAverageDeaths": {
+      "title": "Average death cases in Last 7 Days",
+      "expression": "average(deaths(day-7,day))"
+    },
+    "Last30DaysAverageCases": {
+      "title": "Average cases in last 30 days",
+      "expression": "average(cases(day-30,day))"
+    },
+    "Last30DaysAverageDeaths": {
+      "title": "Average Death cases in last 30 days",
+      "expression": "average(deaths(day-30,day))"
+    },
+    "MaxNewCasesPerDaysoFar": {
+      "title": "Max new cases per day so far",
+      "expression": "max(newcases(first,last))"
+    },
+    "MaxNewDeathsPerDaysoFar": {
+      "title": "Max new death cases per day so far",
+      "expression": "max(newdeaths(first,last))"
+    },
+    "AverageDeathsPerDay": {
+      "title": "Average deaths per day",
+      "expression": "average(deaths(first,last))"
+    },
+    "TotalNewCasesInLast7Days": {
+      "title": "Total new cases in last 7 days",
+      "expression": "cases(day)-cases(day-7)"
+    },
+    "TotalNewCasesInLast30Days": {
+      "title": "Total new cases in last 30 days",
+      "expression": "cases(day)-cases(day-30)"
+    },
+    "TotalNewDeathsInLast7Days": {
+      "title": "Total new death casess in last 7 days",
+      "expression": "deaths(day)-cases(day-7)"
+    },
+    "TotalNewDeathsInLast30Days": {
+      "title": "Total new death cases in last 30 days",
+      "expression": "deaths(day)-cases(day-30)"
+    },
+    "MaxNewCasesinLast7Days": {
+      "title": "Max new cases in last 7 days",
+      "expression": "max(newcases(day-7,day)"
+    },
+    "MaxNewCasesinLast30Days": {
+      "title": "Max new cases in last 30 days",
+      "expression": "max(newcases(day-30,day))"
+    },
+    "MinNewCasesinLast7Days": {
+      "title": "Min new cases in last 7 days",
+      "expression": "min(newcases(day-7,day))"
+    },
+    "MinNewCasesinLast30Days": {
+      "title": "Min new cases in last 30 days",
+      "expression": "min(newcases(day-30,day))"
+    },
+    "TotalRecoveredCasesSoFar": {
+      "title": "Total recovered cases so far",
+      "expression": "sum(newcases(first,last))- sum(cases(first,last))"
+    }
+  },
+  "keywords": {
+    "newcases": {
+      "keyword": "newcases",
+      "description": "total new cases on a day"
+    },
+    "newdeaths": {
+      "keyword": "newdeaths",
+      "description": "total new deaths on a day"
+    },
+    "totalcases": {
+      "keyword": "totalcases",
+      "description": "total cases on a day"
+    },
+    "totalcasesper1M": {
```

Comment:
Words like "totalcasesper1M" and "totaldeathsper1M" aren't keywords. Keywords are things the language recognizes, like newdeaths, newcases, deaths, cases, population, first, last, day, and aggregate operations.

Reply:
Correction done.